code/game/turfs/snow.dm

```diff
@@ -76,19 +76,20 @@
 
 
 /turf/open/floor/plating/ground/snow/Entered(atom/movable/arrived, atom/old_loc, list/atom/old_locs)
-	if(slayer > 0)
-		if(iscarbon(arrived))
-			var/mob/living/carbon/C = arrived
+	if(slayer > 0 && iscarbon(arrived))
+		var/mob/living/carbon/C = arrived
+		if(isxenowarlock(C)) //they float over the snow
```

Comment:
should be a flag

Reply:
I'll just have a var like water, since I can't share the same var for both due to boiler's snowflake